src/components/resizing-textarea.tsx

```diff
@@ -0,0 +1,36 @@
+import Component from 'inferno-component';
+
+export default class ResizingTextarea extends Component<{},{}> {
+
+    textarea : textarea;
```

Comment:
ERROR in ./src/components/resizing-textarea.tsx
    (5,16): error TS2304: Cannot find name 'textarea'.

Should be HTMLTextAreaElement?